documentation/user_docs/docs/conf.py

```diff
@@ -25,6 +25,7 @@
 
 
 # The full version, including alpha/beta/rc tags
+version = '3.5.0'
```

Comment:
these version numbers shouldn't be hard-coded in the the Python.

The `VERSION` file in the main project source tree is the reference version number;  this was created so we didn't have the same version number stored in lots of places across the source. Could this pull it from there? There are routines in the CMake scripts that use this to create the "fully decorated" version number that goes into Matlab and C files.

Reply:
These numbers are overwritten by the build, but true. I could use a CMake template to create this file and not include it in the docs, but that means you can't build the docs without running CMake first (not too unreasonable).

Reply:
That would also mean that I don't need quite as much sed magic.